### DISCUSSION TITLE
Biome v1.4 Rule promotions

### DISCUSSION THREAD

**Discussion by Conaclos** [biomejs/biome#841] [RFC]
## Rule promotion

Biome has a special category named nursery. All rules of this category are unstable and experimental rules.
I propose to promote the following rules to a stable category.

### Accessibility

- [noInteractiveElementToNoninteractiveRole](https://biomejs.dev/linter/rules/no-interactive-element-to-noninteractive-role)
- [useAriaActivedescendantWithTabindex](https://biomejs.dev/linter/rules/use-aria-activedescendant-with-tabindex)

## Complexity

- [noThisInStatic](https://biomejs.dev/linter/rules/no-this-in-static)
- [useArrowFunction](https://biomejs.dev/linter/rules/use-arrow-function)

### Correctness

- [noEmptyCharacterClassInRegex](https://biomejs.dev/linter/rules/no-empty-character-class-in-regex)
- [noInvalidNewBuiltin](https://biomejs.dev/linter/rules/no-invalid-new-builtin)

### Suspicious

- [noMisleadingInstantiator](https://biomejs.dev/linter/rules/no-misleading-instantiator)
- [noMisrefactoredShorthandAssign](https://biomejs.dev/linter/rules/no-misrefactored-shorthand-assign)
- [noApproximativeNumericConstant](https://biomejs.dev/linter/rules/no-approximative-numeric-constant/)

### Style

- [noUselessElse](https://biomejs.dev/linter/rules/no-useless-else)
- [useShorthandAssign](https://biomejs.dev/linter/rules/use-shorthand-assign)
- [useAsConstAssertion](https://biomejs.dev/linter/rules/use-as-const-assertion)

## Rule deprecation

Since we stabilize [noInvalidNewBuiltin](https://biomejs.dev/linter/rules/no-invalid-new-builtin), I propose to deprecate [noNewSymbol](https://biomejs.dev/linter/rules/no-new-symbol).

## Recommended rules

I propose to recommend the folmlowing stable rules:

### Accessibility

- [noAccessKey](https://biomejs.dev/linter/rules/no-access-key/)
- [useHeadingContent](https://biomejs.dev/linter/rules/use-heading-content)

### Compelxity

- [useSimpleNumberKeys](https://biomejs.dev/linter/rules/use-simple-number-keys)

**Comment by ematipico** [comment]
All good! 

Few suggestions:
- we could recommend the other a11y rules (if they aren't)
- it's possible that we don't do anything special when a rule is deprecated (website, and diagnostic), I don't recall. If we don't, we can fix that at the next patch

**Comment by nissy-dev** [comment]
> we could recommend the other a11y rules (if they aren't)

We should recommend [noAccessKey](https://biomejs.dev/linter/rules/no-access-key/) in addition to useHeadingContent.

**Reply by Conaclos** [reply]
Done :)
